src/main/kotlin/org/rust/lang/core/types/ty/TyTypeParameter.kt

```diff
@@ -80,10 +90,17 @@ class TyTypeParameter private constructor(
         fun associated(type: Ty, trait: RsTraitItem, target: RsTypeAlias): TyTypeParameter {
             return TyTypeParameter(
                 AssociatedType(type, trait, target),
-                { emptyList() } )
+                // TODO: support associated type bounds
+                true,
```

Comment:
Bare `true` looks not very readable on GitHub, let's add `isSized = true` for readability?